datalake/src/main/java/com/sequenceiq/datalake/flow/detach/event/SdxStartDetachRecoveryEvent.java

```diff
@@ -0,0 +1,47 @@
+package com.sequenceiq.datalake.flow.detach.event;
+
+import java.util.Objects;
+
+import com.sequenceiq.cloudbreak.common.event.AcceptResult;
+import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.datalake.flow.detach.SdxDetachEvent;
+
+import reactor.rx.Promise;
+
+public class SdxStartDetachRecoveryEvent extends SdxEvent {
```

Comment:
same, please ensure events has `toString` implementation which involves calling the parent class `toString` implementation too

Reply:
There is no parent implementation for that.

Reply:
Added the tostring method though.